docs/PERFORMANCE.md

```
@@ -205,25 +207,29 @@ make -C build regression-test
 
 ## Benchmark Results Reference
 
-### Typical Performance (AMD64, 3.2GHz)
+### Typical Performance (x86_64)
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)